src/files-and-videos/videos-page/VideoThumbnail.jsx

```diff
@@ -26,7 +26,10 @@ const VideoThumbnail = ({
   intl,
 }) => {
   const fileInputControl = useFileInput({
-    onAddFile: (file) => handleAddThumbnail(file, id),
+    onAddFile: (files) => {
+      const [file] = files;
+      handleAddThumbnail(file, id);
+    },
     setSelectedRows: () => {},
     setAddOpen: () => false,
   });
```

Comment:
My philosophy is: when we discover a bug, we should add a test that prevents this bug from re-occurring in the future. If possible - sometimes bugs are extremely hard to test.
Now it seems that a test for this will be extremely easy if you just move this function outside of the component, export it, and then in a test check if `handleAddThumbnail` was called and whether it was called with a single file.
Do you think it makes sense to add a little test for this?

Reply:
I don't think that it makes sense to write a test for this

Reply:
Ah, you mean because we can't test here what format the data is that the backend sends. Good point. I'll resolve the discussion here.
Generally you'd have a test though that expects the backend to send data in a certain format, and then tests that the handling of that data is correct. So in theory there should have been a test previously that expects files to be sent not as array, and with your code change this test should have failed and then would need to be adjusted in this PR. Ideally we'd add such a test but we're already not having full unit test coverage, and this seems to be a less important test than some others.